example/CMakeLists.txt

```diff
@@ -2,19 +2,19 @@
 
 # Subpackage name must match what appears in kokkos/cmake/Dependencies.cmake
 #
-KOKKOS_SUBPACKAGE(Example)
+#KOKKOS_SUBPACKAGE(Example)
 
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(query_device)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(fixture)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(feint)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(fenl)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(multi_fem)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(md_skeleton)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(global_2_local_ids)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(grow_array)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(sort_array)
-if(NOT Kokkos_ENABLE_CUDA)
-KOKKOS_ADD_EXAMPLE_DIRECTORIES(tutorial)
-endif()
-KOKKOS_SUBPACKAGE_POSTPROCESS()
+#KOKKOS_ADD_EXAMPLE_DIRECTORIES(query_device)
+#KOKKOS_ADD_EXAMPLE_DIRECTORIES(fixture)
+#KOKKOS_ADD_EXAMPLE_DIRECTORIES(feint)
+#KOKKOS_ADD_EXAMPLE_DIRECTORIES(fenl)
+#KOKKOS_ADD_EXAMPLE_DIRECTORIES(multi_fem)
+#KOKKOS_ADD_EXAMPLE_DIRECTORIES(md_skeleton)
+#KOKKOS_ADD_EXAMPLE_DIRECTORIES(global_2_local_ids)
+#KOKKOS_ADD_EXAMPLE_DIRECTORIES(grow_array)
+#KOKKOS_ADD_EXAMPLE_DIRECTORIES(sort_array)
+#if(NOT Kokkos_ENABLE_CUDA)
+#KOKKOS_ADD_EXAMPLE_DIRECTORIES(tutorial)
+#endif()
+#KOKKOS_SUBPACKAGE_POSTPROCESS()
```

Comment:
I am still not quite sure I understand why just commenting (especially the examples already deleted in this pull request) is the best approach. Why not just delete the file and all the other `CMakeLists.txt` files altogether?